Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -395,8 +395,8 @@ - (void)launchApplicationWithContext:(BPExecutionContext *)context {
         NEXT([__self deleteSimulatorWithContext:context andStatus:BPExitStatusLaunchAppFailed]);
     };
 
-    [context.runner launchApplicationAndExecuteTestsWithParser:context.parser andCompletion:handler.defaultHandlerBlock isHostApp:NO];
-    [BPUtils printInfo:INFO withString:[@"Yay, after the launch async call!!!" stringByAppendingString:stepName]];
+    [context.runner launchApplicationAndExecuteTestsWithParser:context.parser andCompletion:handler.defaultHandlerBlock];
+    [BPUtils printInfo:INFO withString:[@"Async Launch Returned" stringByAppendingString:stepName]];
```

Comment:
Thanks for that. That text wasn't supposed to remain. I just forgot to remove it.... :)